tests/src/Traits/EntityUtilsTrait.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/**
+ * Copyright 2018 Google Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License version 2 as published by the
+ * Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc., 51
+ * Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ */
+
+namespace Drupal\Tests\apigee_edge\Traits;
+
+use Drupal\Core\Url;
+
+/**
+ * A trait to common functions of Apigee Edge entity tests.
+ */
+trait EntityUtilsTrait {
+
+  /**
+   * Changes and validates the singular and plural aliases of the entity.
+   *
+   * @param string $entity_type_id
+   *   The ID of the entity type.
+   * @param string $entity_settings_route_name
+   *   The name of the entity settings route.
+   */
+  protected function changeEntityAliasesAndValidate(string $entity_type_id, string $entity_settings_route_name) {
```

Comment:
Probably this is not the scope, but I think later we should not just change the labels of these entity in one tests rather change them in the apigee_edge_testing module and use the assertions in every tests to ensure the aliased entity label gets displayed everywhere where the given entity's label gets displayed.

Reply:
Agreed!